datasets/cmip6-tas.data.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_DEMO
-name: 'CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset)'
+name: 'CMIP6 Daily GISS-E2-1-G Near-Surface Air Temperature (demo subset; 1950-2014)'
```

Comment:
Does this still need to include `demo subset`? Couldn't we just say "Historical CMIP6..."?